nav-app/src/app/viewmodels.service.ts

```diff
@@ -1130,6 +1130,7 @@ export class ViewModel {
             else console.error("TypeError: sorting field is not a number")
         }
         if ("hideDisabled" in obj) {
+            obj.hideDisabled = JSON.parse(obj.hideDisabled);
```

Comment:
I'm curious why this (and the below uses of `JSON.parse` for booleans) is nessecary? `deSerialize` already parses the entire structure, so this would only be useful if the boolean here was a string (`"false"`) instead of an actual boolean (`false`). "String booleans" aren't technically allowed in the Layer spec so I don't see why it should be added here unless there's something I'm missing.